VAQR.iml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I wonder why this file is here. Try deleting it and commit again. If it appears again next time, add `*.iml` in the .gitignore file